docs/build/isc/v1.3/docs/how-tos/core-contracts/nft/mint-nft.md

```diff
@@ -19,12 +19,20 @@ The Stardust update allows you to create your own NFTs. You can also use [IRC27]
 
 2. Get the senders AgentID:
 
+The `ISCAgentID` represents the identifier of the agent (user or contract) whose NFTs you want to retrieve. You can get the `AgentID` from the sender by calling `ISC.sandbox.getSenderAccount()`.
+
 ```solidity
 ISCAgentID memory agentID = ISC.sandbox.getSenderAccount();
 ```
 
 3. Create an `IRC27Metadata` struct  with all the needed data:
 
+:::tip 
+You can refer to [Get NFT Metadata guide](/isc/how-tos/core-contracts/nft/get-nft-metadata/) to understand how to create an `IRC27Metadata` 
```

Comment:
Hi @vivekjain23, The URL here differs from the one in v1.1 above that Lucas suggested. That may be what is causing the last test not to pass. Here, you are providing a full path, but in v1.1, you provided this `get-nft-metadata.md`